app/_src/policies/introduction.md

```diff
@@ -64,14 +64,63 @@ The `spec` field contains the actual configuration of the policy.
 
 Some policies apply to only a subset of the configuration of the proxy.
 
-- **Inbound policies** apply only to incoming traffic. The `spec.from[].targetRef` field defines the subset of clients that are going to be impacted by this policy.
-- **Outbound policies** apply only to outgoing traffic. The `spec.to[].targetRef` field defines the outbounds that are going to be impacted by this policy
+{% if_version gte:2.10.x %}
+
+- **Inbound policies** apply only to incoming traffic. Most inbound policies now use `spec.rules[]` to define their
+  configuration.
+  However, [`MeshTrafficPermission`](../meshtrafficpermission) and [`MeshFaultInjection`](../meshfaultinjection) still
+  use the `spec.from[].targetRef` field,
+  which defines the subset of clients that are going to be impacted by this policy.
+- **Outbound policies** apply only to outgoing traffic. The `spec.to[].targetRef` field defines the outbounds that are
+  going to be impacted by this policy
+{% endif_version %}
+{% if_version eq:2.9.x %}
+- **Inbound policies** apply only to incoming traffic. The `spec.from[].targetRef` field defines the subset of clients
+  that are going to be impacted by this policy.
+- **Outbound policies** apply only to outgoing traffic. The `spec.to[].targetRef` field defines the outbounds that are
+  going to be impacted by this policy
+{% endif_version %}
 
 The actual configuration is defined under the `default` field.
 
 For example:
 
+{% if_version gte:2.10.x %}
+{% policy_yaml %}
+
+```yaml
+type: ExampleOutboundPolicy
+name: my-example
+mesh: default
+spec:
+  targetRef:
+    kind: Mesh # policy applies to all proxies in the mesh
+  to:
+    - targetRef:
+        kind: MeshService # only for requests destined for 'my-service'
+        name: my-service
+      default: # configuration that applies to selected requests on selected proxies
+        key: value
+---
+type: ExampleInboundPolicy
+name: my-example
+mesh: default
+spec:
+  targetRef:
```

Comment:
I am not sure if we should introduce Dataplane kind here, as it can be also used for outbound policies. I think we should stick with Mesh

Reply:
yeah but the example above `ExampleOutboundPolicy` already shows `targetRef.kind: Mesh`, so I think it makes sense to show all possible variations here